eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -195,7 +208,19 @@ function New-ChangeLogEntry {
     return $null
   }
 
-  if (!$Content) { $Content = @() }
+  if (!$Content) { 
```

Comment:
@chidozieononiwu one other question that I just thought about is now our verify changelog check that validates that the change log isn't empty should be updated to verify that the change log entry doesn't have all empty sections.